Added shell script to install python dependencies

bsrinivasa opened the pull request:
Python dependencies required during setup of the tool for the first time. 

cc: @amishas157

bsrinivasa committed:
Added shell script to install python dependencies

bsrinivasa committed:
Added shell script to setup python dev

bsrinivasa commented:
Added shell script to setup python dev version on linux machine if the python version is not higher than 2.6

bsrinivasa committed:
Added example for osm.py

amishas157 commented on setup.sh:
```diff
@@ -0,0 +1,4 @@
+#python dependencies required for the tool
```
We can make a requirements file for this. For example: https://github.com/mapbox/wikimama/blob/master/requirements.txt

amishas157 commented:
@bsrinivasa Thanks for the PR, It looks all good to me. Let's merge after getting the requirements file and adding docs for using these scripts in README.md ?

bsrinivasa committed:
Added requirements.txt to install python dependencies, removed setup.sh

bsrinivasa commented:
@amishas157 have updated the python package dependencies in a text file. Here are the new changes.
- Added python dependencies with appropriate version numbers to `requirements.txt`
- Updated README.md with the instructions supporting ^.

bsrinivasa committed:
typo fix

amishas157 commented:
Yay, Awesome. Thanks
